py_mmd_tools/nc_to_mmd.py

```diff
@@ -1133,8 +1133,8 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
 
         This list can be extended but requires some new code...
         """
-        if collection is not None and type(collection) not in [list, str]:
-            raise ValueError('collection must be of type str or list')
+        if collection is not None and type(collection) is not str:
+            raise ValueError('collection must be of type str')
```

Comment:
```suggestion
            raise ValueError('collection must be given and of type str')
```

Reply:
Look closely at 1136 - it does not need to be given..